lib/features/email/data/repository/calendar_event_repository_impl.dart

```diff
@@ -49,4 +53,24 @@ class CalendarEventRepositoryImpl extends CalendarEventRepository {
     return _calendarEventDataSource[DataSourceType.network]!
       .rejectEventInvitation(accountId, blobIds, language);
   }
+
+  @override
+  Future<List<BlobCalendarEvent>> transformCalendarEventDescription(
+    List<BlobCalendarEvent> blobCalendarEvents,
+    TransformConfiguration transformConfiguration,
+  ) async {
+    return Future.wait(blobCalendarEvents.map((blobCalendarEvent) async {
+      return BlobCalendarEvent(
+        blobId: blobCalendarEvent.blobId,
+        calendarEventList: await Future.wait(blobCalendarEvent.calendarEventList.map((calendarEvent) async {
+          return calendarEvent.copyWith(
+            description: await _htmlDataSource.transformHtmlEmailContent(
```

Comment:
why do we need `async` function?

Reply:
Because `await _htmlDataSource.transformHtmlEmailContent` is `Future`